### DISCUSSION TITLE
Zoom In and Zoom Out not working for me

### DISCUSSION THREAD

**Discussion by Ahmed-142** [apache/netbeans#6809] [Q&A] [answered]
Hello,

I have Apache NetBeans IDE 20.

I want to zoom out the screen.

In NetBeans, I went to View>Toolbars>Customize
Then, I dragged "Zoom In" and "Zoom Out" items.

Then, in the toolbar I become able to see the icons of Zoom in and Zoom out, but when I click on Zoom in or Zoom out nothing happens, nothing is zoomed out or zoomed in.

![image](https://github.com/apache/netbeans/assets/153163090/460ab56b-5748-45e1-8078-fa4d5f19d510)

**Comment by neilcsmith-net** [comment]
Those zoom actions appear to be the ones from the image viewer.  Open an image file and you can zoom in and out with them.

There is a zoom action for the text in the editor that uses `ALT+MOUSE_WHEEL` - that does not seem to be an action that can be added to the toolbar.  There isn't a zoom action covering all of the UI.

**Reply by Ahmed-142** [reply]
Hi [neilcsmith-net](https://github.com/neilcsmith-net),

Thank you for your reply.

Actually, it seems that my mouse does not work, and I usually use touchpad on my laptop instead of mouse.

Is there any other way to zoom in and zoom out the text other than the way of using Alt key and mouse wheel?

**Reply by neilcsmith-net** [reply, marked as answer]
If you can scroll using the touchpad, I think this should still work when holding `Alt`?

**Reply by Ahmed-142** [reply]
I tried to do that, first time I tried to hold pressing on Alt key and trying to do the action of zooming out and zooming in but it becomes smaller and smaller so it was zooming out and I coudn't zoom in. After that, I closed NetBeans and I open it again so I think it's in the default zoom position or in the default zoom setting, then I tried again and I can zoom in but it seems even when I try to zoom out it zooms in.

Anyway, I just increased font size to 18, and that seems ok for me.

**Reply by Chris2011** [reply]
Hey @Ahmed-142 I also use touchpad the whole time and alt + gesture should work for both directions. If not, please try to identify he exact problem when does this happen, etc.
